Add missing meta descriptions (#488)
mongodb/docs-golang: source/usage-examples/insertOne.txt

Insert a Document Example
=========================

.. meta::

>    :description: Learn how to insert a document into a MongoDB collection using the `InsertOne()` method in Go, with an example using a `Restaurant` struct.

Reviewer: ```suggestion
   :description: Learn how to insert a document into a collection using the insertOne() method in the MongoDB Go Driver.
```
